Add OKTETO_SMART_BUILDS_IGNORE_FILES_ENABLED to feature flags (#983)
okteto/docs: src/content/reference/feature-flags.mdx

| OKTETO_KUBERNETES_TIMEOUT                        | Specifies the timeout while deploying                                                                                                                                                                                           | `0`                    |
| OKTETO_LOCAL_REGISTRY_STORE_PRIORITY_ENABLED     | Gives more priority to the local registry credentials than the Okteto Registry Credentials                                                                                                                                      | `false`                |
| OKTETO_SMART_BUILDS_ENABLED                      | Enables the [Smart Builds feature](core/build-service.mdx#smart-builds) that uses a hash of the build context to determine if an image needs to be rebuilt or can be pulled from the registry. This can accelerate build times  | `true`                 |

> | OKTETO_SMART_BUILDS_IGNORE_FILES_ENABLED         | If set, considers ignores rules in .dockerignore for calculating smart build context.                                                                                                                         | `false`                |

Reviewer: ```suggestion
| OKTETO_SMART_BUILDS_IGNORE_FILES_ENABLED         | When enabled, files listed in `.dockerignore` will be excluded from the smart build context calculation                                                                                                                         | `false`                |
```

> | OKTETO_SMART_BUILDS_IGNORE_FILES_ENABLED         | If set, considers ignores rules in .oktetoignore and .dockerignore for calculating smart build context.                                                                                                                         | `false`                |

Reviewer 2: why do we consider `.oktetoignore`? smart builds only works with `dockerignore` files, no?

Reviewer 2: is there a valid case for `.oktetoignore`? I think it's a bug if we are considering it, but maybe I am missing something
